CanvasColorSpaceProposal.md

```diff
@@ -1,52 +1,29 @@
-* The bit-depth of canvases is currently fixed to 8 bits per component, which is below the capabilities of some monitors. Monitors with higher contrast ratios require more bits per component to avoid banding.
-* Color encoding and blending is de facto perceptually-linear unorm8, which allocates too little precision on low values (and too much precision on high values), leading to poor handling of dark scenes and content.
-
-### Current Usage and Workarounds
-The lack of color space interoperability is hard to work around. For some browser implementations which color-correct images drawn to canvases by applying the display profile, apps that want to use canvases for color corrected image processing are stuck doing convoluted workarounds, such as:
-* reverse-engineer the display profile by drawing test pattern images to the canvas and inspecting the color corrected result via getImageData
-* bypass CanvasRenderingContext2D.drawImage() and use image decoders implemented in JavaScript to extract raw image data that was not tainted by the browser's color correction behavior.''
-
-An aspect of current implementations that is interoperable is that colors match between CSS/HTML and canvases:
-* A color value used as a canvas drawing style will have the same appearance as if the same color value were used as a CSS style
-* An image resource drawn to a canvas element will have the same appearance as if it were displayed as the replaced content of an HTML element or used as a CSS style value.
-
-This color matching behavior needs to be preserved to avoid breaking pre-existing content.
-
-Some implementations convert images drawn to canvases to the sRGB color space. This has the advantage of making the color correction behavior device independent, but it clamps the gamuts of the rendered content to the sRGB gamut, which is significantly narrower than the gamuts of some current consumer devices.
+* The color space of canvases is undefined in the current specification, although is de facto sRGB.
+* The bit-depth of canvases is likewise undefined, although is de facto 8 bits per component.
+* Images retrieved from a canvas are in an undefined color space and no color space information is encoded by ``toDataURL`` or ``toBlob``.
 
 ### Requests for this Feature
 
 * <cite>[https://github.com/whatwg/html/issues/299]</cite> <blockquote><p>Allow 2dcontexts to use deeper color buffers</p></blockquote>
 * <cite>[https://bugs.chromium.org/p/chromium/issues/detail?id=425935]</cite> <blockquote><p>Wrong color profile with 2D canvas</p></blockquote>
-* Engineers from the Google Photos, Maps and Sheets teams have expressed a desire for canvases to become color managed.  Particularly for the use case of resizing an image, using a canvas, prior to uploading it to the server, to save bandwidth. The problem is that the images retrieved from a canvas are in an undefined color space and no color space information is encoded by toDataURL or toBlob.
```

Comment:
It's unfortunate to lose this bit of feedback; I think it should be left in.

Reply:
Re-added.